src/compiler/checker.ts

```diff
@@ -13041,6 +13041,16 @@ namespace ts {
                                     // JsxAttributes has an object-literal flag and undergo same type-assignablity check as normal object-literal.
                                     // However, using an object-literal error message will be very confusing to the users so we give different a message.
                                     // TODO: Spelling suggestions for excess jsx attributes (needs new diagnostic messages)
+
+                                    if (errorNode && isJsxOpeningLikeElement(errorNode.parent)) {
+                                        const attributes = errorNode.parent.attributes;
+                                        for (const jsxProperty of attributes.properties) {
+                                            if (jsxProperty.kind === SyntaxKind.JsxAttribute && jsxProperty.name.escapedText === prop.escapedName) {

```

Comment:
If you still end up having code like this, switch from `jsxProperty` to `attribute` or something similar.

Reply:
Removed this so the name is no longer relevant